### DISCUSSION TITLE
Is there an alias for the system font?

### DISCUSSION THREAD

**Discussion by jrmoulton** [slint-ui/slint#693] [Q&A] [answered]
In CSS it's pretty common to have a the fonts set like this ```-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"```. In my use case this is especially useful for having the colored emojis rather than just the plain ones but if I set ```"Apple Color Emoji"``` then I also have to specify another font for everything else but that then is not platform independent. 
So is there an alias that I can add to ```font-family``` so that I can do something like ```"-system-font, Apple Color Emoji,..."```?

**Comment by tronical** [comment, marked as answer]
Unfortunately there's no such alias right now, but this is a great suggestion. I'm inclined to say that we should stick to the generic CSS font names, where emojis then are covered with "emoji". In a way this already works to some extend (not "emoji", but "sans-serif"), but we need to document it and make sure it works with all backends.
I'll go ahead and convert this into an issue for tracking.

On a side note: Using emojis should just work. We try hard to find a font with emojis if you're using them even if the font-family is not set to an emoji font.

**Reply by jrmoulton** [reply]
Thanks! 
Emojis are found when I'm using the default font but they aren't colored emojis. Setting other emoji fonts will allow for colored emojis to be used just like in CSS.

**Reply by tronical** [reply]
Good point! Do you see this on windows, btw?

**Reply by jrmoulton** [reply]
I'm not sure on windows. Actually it looks like colored emojis are apple specific. I was just looking at github src and it looks like if the colored emoji isn't avaliable github will just use an image instead. ```<g-emoji class="g-emoji" alias="globe_with_meridians" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f310.png">🌐</g-emoji>```
which is what it is doing on ubuntu but on macOS it renders a colored unicode emoji. So I guess the colored emojis are more typically just for macOS use cases.
